src/components/Tab/src/Tab.tsx

```diff
@@ -47,7 +49,15 @@ export interface TabProps extends Omit<BaseProps, 'classes'> {
  * @constructor Construct an instance of Tab.
  */
 export const Tab: React.FC<TabProps> = (props: TabProps) => {
-  return <MaterialTab {...props} />;
+  const rootClassNames = clsx('denhaag-tabs__tab', props.className);
+  return (
+    <MaterialTab
+      {...props}
```

Comment:
For now I would say this is fine, not sure about what we decided in the `{...props}` discussion though. I believe we said we would use `{...props}` for our own-built components, but specificy all `props` individually for the MUI components due to us not wanting to give access to all MaterialUI props for non-typescript users (correct me if I'm wrong).

Reply:
Indeed that was our conclusion. As discussed during the standup we'll leave it as is for this component, since it'll be rewritten to a custom component soon, and as we need to add support for the HTMLAttributes, explicitly passing every prop seems overkill. (Does that correctly summarise our conclusion?)